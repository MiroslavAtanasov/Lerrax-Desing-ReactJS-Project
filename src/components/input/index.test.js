import React from 'react'
import renderer from 'react-test-renderer'
import Input from '.'
import TestingEnvironment from '../../test-utils/router'

describe('Input Component', () => {
    it('should render Input component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Input />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})