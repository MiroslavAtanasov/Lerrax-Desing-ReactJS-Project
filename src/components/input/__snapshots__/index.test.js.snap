// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component should render Input component 1`] = `
<div
  className="undefined-container"
>
  <div
    className="undefined-inner-container"
  >
    <label
      className="undefined-label"
    />
  </div>
  <input
    className="undefined-input"
    type="text"
  />
</div>
`;
